### PR TITLE
Feat/pd 2996

### PR DESCRIPTION
**Pull request by arimieandreea**

### CODE REVIEW

File: packages/categorize/configure/src/design/categories/index.jsx [hunk 1]
```diff
@@ -136,14 +136,17 @@ export class Categories extends React.Component {
 
   addChoiceToCategory = (addedChoice, categoryId) => {
     const { model, onModelChanged } = this.props;
-    let { choices = [], correctResponse = [] } = model || {};
+    console.log('Andreea model', model);
```
Comment (PatriciaRomaniuc) [inline packages/categorize/configure/src/design/categories/index.jsx:139]: remove console log

File: packages/categorize/configure/src/design/choices/__tests__/choice.test.jsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ describe('choice', () => {
   let connectDropTarget;
 
   beforeEach(() => {
+    console.log('aleluia');
```
Comment (PatriciaRomaniuc) [inline packages/categorize/configure/src/design/choices/__tests__/choice.test.jsx:13]: remove console log

File: packages/categorize/configure/src/design/index.jsx [hunk 3]
```diff
@@ -254,6 +273,9 @@ export class Design extends React.Component {
       rubricEnabled: withRubric?.settings && toggle(withRubric?.label),
     };
 
+    const isOpened = this.isAlertModalOpened();
+    const alertMaxChoicesMsg = `To change this value to ${maxChoicesPerCategory}, each category must have ${maxChoicesPerCategory} or fewer answer choice[s]`;
```
Comment (PatriciaRomaniuc) [inline packages/categorize/configure/src/design/index.jsx:277]: move this into translate in pie-lib

File: packages/categorize/configure/src/utils.js [hunk 4]
```diff
@@ -18,3 +18,11 @@ export const generateValidationMessage = (config) => {
 
 // used in controller too, for consistency modify it there too
 export const multiplePlacements = { enabled: 'Yes', disabled: 'No', perChoice: 'Set Per Choice' };
+
+export const  getMaxCategoryChoices = (model) => {
+  const { correctResponse } = model || [];
+  return correctResponse.reduce((max, correctRes) => {
```
Comment (PatriciaRomaniuc) [inline packages/categorize/configure/src/utils.js:24]: add some sanity checks inside this method for correctResponse

File: packages/categorize/src/categorize/index.jsx [hunk 5]
```diff
@@ -151,6 +189,7 @@ export class Categorize extends React.Component {
     const existAlternate = this.existAlternateResponse(correctResponse) || false;
     const displayNote =
       (showCorrect || (mode === 'view' && role === 'instructor')) && showNote && note && existAlternate;
+    const alertMessage = `You ve reached the limit of ${maxChoicesPerCategory} responses per area. To add another response, one must first be removed`;
```
Comment (PatriciaRomaniuc) [inline packages/categorize/src/categorize/index.jsx:192]: same here, add translations

**Review by PatriciaRomaniuc — CHANGES_REQUESTED** [review]

**Comment by arimieandreea** [comment]
https://illuminate.atlassian.net/jira/software/c/projects/PD/issues/PD-2996

**Pull request merged**